data_model/src/trigger/mod.rs

```diff
@@ -226,6 +230,8 @@ pub mod action {
         /// account must already exist in order for `Register<Trigger>` to
         /// work.
         pub technical_account: crate::account::Id,
+        /// Domain linked to this trigger.
+        pub domain: Option<crate::domain::Id>,
```

Comment:
I like the idea proposed by @appetrosyan in the chat. Let's use *id* part of the trigger to identify if it's a domain-scoped trigger or not